README.md

```diff
@@ -196,6 +196,22 @@ python3 serve_google_api.py --endpoint localhost:50051 --checkpoint ...
 python3 scripts/stt_google.py --endpoint localhost:50051 --lang ru --api-key-credentials= -i calls_micro/calls_micro.json
 ```
 
+# Frontend performance metrics
+```shell
+bash scripts/read_audio_performance.sh
+```
+| file       | reads count |  backend | process_time ms| perf_counter ms| 
```

Comment:
We discussed that it should be microseconds. "ms" is milliseconds. microseconds is "us"

Reply:
done